packages/analytics/matomo/usePaq.ts

```diff
@@ -1,6 +1,14 @@
+declare global {
+   interface Window {
+      _paq: any;
+   }
+}
+
 export function usePaq(): any[] | undefined {
    if (typeof window === 'undefined') {
       return undefined;
    }
-   return (window as any)._paq;
+
+   var _paq = (window._paq = window._paq || []);
```

Comment:
Oh wait... this may be troublesome cause callers may capture this value in a closure (it's really easy to do accidentally).

I'm tempted to say we should return a `pushToPaq(args: array): void` function that will do this dynamic lookup each time it's called. Thoughts?